tidy3d/components/simulation.py

```diff
@@ -1378,11 +1368,17 @@ def grid(self) -> Grid:
         :class:`Grid`
             :class:`Grid` storing the spatial locations relevant to the simulation.
         """
-        cell_boundary_dict = {}
-        for dim, key in enumerate("xyz"):
-            cell_boundary_dict[key] = self._make_bound_coords(dim)
-        boundaries = Coords(**cell_boundary_dict)
-        return Grid(boundaries=boundaries)
+
+        # Add a simulation Box as the first structure
+        structures = [Structure(geometry=self.geometry, medium=self.medium)]
```

Comment:
Actually, this doesn't work, because the box in that property has infinite size. Here we actually make a structure with the Simulation.geometry, which is used in make_grid. I think there's a need for both definitions of the backgroudn.

Reply:
k, yea let's make it a property then?

Reply:
maybe `self.structure` (singular?)

Reply:
I'm a bit worried about future confusion with background_structure.

Reply:
ok feel free to rename these or leave one of them out.